packages/nivo-bar/index.d.ts

```diff
@@ -0,0 +1,149 @@
+declare module '@nivo/bar' {
+    export class Bar extends React.Component<Data & BarProps & Dimensions>{}
+    export class BarCanvas extends React.Component<Data & BarProps & Dimensions>{}
+    export class ResponsiveBar extends React.Component<Data & BarProps>{}
+    export class ResponsiveBarCanvas extends React.Component<Data & BarProps>{}
+
+    export type Data = {
+        data: object[];
+    }
+
+    export type BarProps = Partial<{
+        indexBy: string | Function;
+        keys: string[];
+
+        groupMode: 'stacked' | 'grouped';
+        layout: 'horizontal' | 'vertical';
+        reverse: boolean;
+
+        innerPadding: number;
+        minValue: number | 'auto';
+        margin: Box;
+        maxValue: number | 'auto';
+        padding: number;
+
+        // axes & grid
+        axisBottom: Axis;
+        axisLeft: Axis;
+        axisRight: Axis;
+        axisTop: Axis;
+        enableGridX: boolean;
+        enableGridY: boolean;
+
+        // customization
+        barComponent: Function;
+        colors: string | string[] | Function;
+        colorBy: string | Function;
+
+        // labels
+        enableLabel: boolean;
+        label: string | Function;
+        labelFormat: string | Function;
+        labelLinkColor: string | Function;
+        labelSkipWidth: number;
+        labelSkipHeight: number;
+        labelTextColor: string | Function;
+
+        // theming
+        borderRadius: number;
+        borderWidth: number;
+        defs: Array<{ id: string }>;
+        fill: Array<{ id?: string, match: object | Function | '*' }>;
+        theme: Theme;
+
+        // interactivity
+        isInteractive: boolean;
+        onClick: Function;
+        tooltip: Function;
+        tooltipFormat: string | Function;
+
+        // motion
+        animate: boolean;
+        motionDamping: number;
+        motionStiffness: number;
+
+        legends: Array<{ dataFrom: 'indexes' | 'keys' } & Legend>;
+
+        // canvas specific
+        pixelRatio: number;
+    }>
+
+    export type Axis = Partial<{
+        format: string | Function;
+        legend: string;
+        legendOffset: number;
+        legendPosition: 'start' | 'center' | 'end';
```

Comment:
I'm currently `ResponsiveBar` and `legendPosition: 'center'` aligns the label off the canvas. However, `legendPosition: 'middle'` works as expected, but doesn’t pass TypeScript validation? Which is correct.

There is also no property for renderTick should this be exposed?